requirements.txt

```diff
@@ -4,4 +4,4 @@ flask-SQLAlchemy==2.3.1
 psycopg2-binary==2.7.5
 graphene[sqlalchemy]
 Flask-GraphQL==2.0.0
-
+sqlacodegen==2.0.0
```

Comment:
Where is this used in the project? Is to generate the models? 

If so please make a wiki showing how to use as a reference.

Reply:
https://github.com/agronholm/sqlacodegen